### ISSUE TITLE
battery percentage is not displaying big-sur

### ISSUE THREAD

**Issue by todd-dsm** [todd-dsm/mac-ops#39]
###---                                                                              
### Show battery percentage                                                         
###---                                                                              
printInfo "Show battery percentage..."                                              
# defaults read com.apple.menuextra.battery ShowPercent                             
defaults write com.apple.menuextra.battery ShowPercent -string 'YES'

**Comment by tkothe** [comment]
yeah, I'd also like to know where that setting is stored in Big Sur now... 
I was able to manually change it in the System Preferences UI, but no luck yet finding what underlying `defaults write` setting it changed.

**Comment by KostyaEsmukov** [comment]
`defaults write ~/Library/Preferences/ByHost/com.apple.controlcenter.plist BatteryShowPercentage -bool true` seems to work in Big Sur.

**Comment by todd-dsm** [comment]
@KostyaEsmukov just tested on the latest Big Sur (11.6.1 (20G224)) and it works as you described it. I'll close this issue once it's coded up on the Big Sur branch.

Thank you!
